warehouse/query-core/src/main/java/datawave/query/iterator/ResultCountingIterator.java

```diff
@@ -43,9 +44,9 @@ public boolean hasNext() {
     }
     
     private Key addKeyCount(Key key) {
-        resultCount++;
```

Comment:
[ERROR] Increment of volatile field datawave.query.iterator.ResultCountingIterator.resultCount in datawave.query.iterator.ResultCountingIterator.addKeyCount(Key) [datawave.query.iterator.ResultCountingIterator] At ResultCountingIterator.java:[line 46] VO_VOLATILE_INCREMENT